subprojects/docs/src/docs/release/notes.md

```diff
@@ -301,6 +301,18 @@ The same applies to builds run through TestKit.
 Gradle 5.0 requires a minimum Tooling API client version of 3.0. 
 Older client libraries can no longer run builds with Gradle 5.0
 
+### Lower default memory settings
+
+The command line client now starts with 64m of heap instead of 1g. 
```

Comment:
I think it'd be very helpful to include this in the [upgrade_gradle_4.adoc](https://docs.gradle.org/5.0-milestone-1/userguide/upgrading_version_4.html). 

Seems like it ought to get it's own bullet point in the preamble. Agreed @pledbrook?

Reply:
Indeed.